include/mbedtls/config.h

```diff
@@ -48,10 +48,14 @@
  * Requires support for asm() in compiler.
  *
  * Used in:
+ *      library/aria.c
  *      library/timing.c
- *      library/padlock.c
```

Comment:
That's intentional: the "Used in" list if for modules that will use it if available but will work normally otherwise (in that case, typically because there is a pure-C fallback). The "Required by" list is for things that won't compile without it: every entry from that list should have a corresponding check in `check_config.h`.

Padlock was misplaced so I fixed that while I was editing the list to add Aria.